hack/local-up-kubeedge.sh

```diff
@@ -278,6 +275,12 @@ generate_streamserver_cert
 
 start_cloudcore
 
+# install CNI plugins
+if [[ "${CONTAINER_RUNTIME}" = "remote" ]]; then
+# we need to install CNI plugins only when we use remote(containerd) as edgecore container runtime
+install_cni_plugins ${CNI_DOWNLOAD_ADDR} ${CNI_CONF_OVERWRITE}
```

Comment:
The `CNI_DOWNLOAD_ADDR` is nil now, if we set `CNI_CONF_OVERWRITE` to true, in `install_cni_plugins`, `CNI_DOWNLOAD_ADDR` will be true.

As the `CNI_DOWNLOAD_ADDR` and `CNI_CONF_OVERWRITE` all global value, it's no need to set as parameter here?

Reply:
Yes，i removed the CNI_DOWNLOAD_ADDR  and CNI_CONF_OVERWRITE variables from local-up-kubeedge.sh.  The install_cni_plugins function will get them directly from the global values.